static/src/js/util/request/granuleRequest.js

```diff
@@ -75,29 +73,12 @@ export default class GranuleRequest extends CmrRequest {
 
       if (id) {
         // Retrieve collection thumbnail if it exists
```

Comment:
Can we drop this comment it does not seem like its valid anymore (or really ever was?)